project/Build.scala

```diff
@@ -35,6 +35,12 @@ object ScalaMeterBuild extends MechaRepoBuild {
 
   val releasePluginSettings =  releaseSettings ++ Seq(
     releaseBranchName := s"version/${(version in ThisBuild).value}",
+    examples.repo := "git@github.com:scalameter/scalameter-examples.git",
+    examples.tag := "v%s",
+    examples.tagComment := "Release %s",
+    examples.commitMessage := "Set ScalaMeter version to %s",
+    examples.scalaMeterVersionFile := "version.sbt",
```

Comment:
Should this be named something like "scalameterVersion.sbt"?
And should the project definitions in `scalameter-examples` also change their `libraryDependencies` setting to use the content in this file?